op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -59,25 +48,58 @@ var (
 		Usage:    "Path to write the rollup config JSON file, to be provided to op-node with the 'rollup.config' flag",
 		Required: true,
 	}
-	dbPathFlag = &cli.StringFlag{
-		Name:     "db-path",
-		Usage:    "Path to the Celo database, not including the `celo/chaindata` part",
+	oldDBPathFlag = &cli.PathFlag{
+		Name:     "old-db",
+		Usage:    "Path to the old Celo chaindata",
+		Required: true,
+	}
+	newDBPathFlag = &cli.PathFlag{
+		Name:     "new-db",
+		Usage:    "Path to write migrated Celo chaindata",
 		Required: true,
 	}
-	dryRunFlag = &cli.BoolFlag{
-		Name:  "dry-run",
-		Usage: "Dry run the upgrade by not committing the database",
+	batchSizeFlag = &cli.Uint64Flag{
+		Name:  "batch-size",
+		Usage: "batch size to use for block migration",
+		Value: 50000,
+	}
+	memoryLimitFlag = &cli.Int64Flag{
+		Name:  "memory-limit",
+		Usage: "Memory limit in MB",
```

Comment:
```suggestion
		Usage: "Memory limit in MB, should be set lower than the available amount of memory in your system to prevent out of memory errors",
```